api/main.py

```diff
@@ -35,20 +35,25 @@ async def ask(query:str, n: Union[int,None] = None) -> ApiResponse[AskResponse]:
     return ApiResponse.create(response)
 
 @app.get('/briefs', tags=tags)
-async def briefs(email:str, start:datetime.datetime, end: Union[datetime.datetime, None] = None, certainty: Union[float,None] = None) -> ApiResponse[BriefResponse]:
+async def briefs(email:str, start:datetime.datetime, end: Union[datetime.datetime, None] = None, certainty: Union[float,None] = None, threshold: Union[float,None] = None) -> ApiResponse[BriefResponse]:
     """Create briefings for a user."""
     if certainty is not None:
         if certainty < 0 or certainty > 100:
             APISupport.error_response(400, "Confidence must be a percentage value between 0 and 100.")
         else:
             certainty = certainty / 100
+    if threshold is not None:
```

Comment:
This will get ugly over time, any time you copy paste you should consider what it will do to the code in the future. A function like `def check_percentage(value: Union[float, None], field_name:str) -> None` could handle both of these